app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -108,6 +115,17 @@ class Exposure extends React.Component {
         }
       }
       this.updateLDEandCEValidations({ ...current.patient, [field]: date });
+    } else if (field === 'symptom_onset') {
+      // turn off NSH if SO is populated
+      if (date) {
+        current.patient.no_symptom_history = false;
```

Comment:
Since setting SO results in setting `no_symptom_history` to `false` in enrollment, I think we'd also want the same behavior in when modifying SO after enrollment. Right now if I set SO on a monitoree page, `no_symptom_history` remains `true`. I guess the place to do that would be in `SymptomOnset.js`, but it also feels like that should be a side effect that we handle in the back end. What do you think?

Reply:
My immediate thought would be to make the behavior consistent/similar to how the LDE/CE logic is handled and the last time I checked, that logic is handled on the front end. However, I do think it would make more sense for this to be handled/validated on the back end as well. I'm wondering if the PO team would want to notify the user that 'No Symptom History' is a field that is turned off when SO date is manually populated by a user.

Although there are similarities between the SO/NSH logic and the LDE/CE logic, there are a few differences as well, one of them being that NSH can (and should) be recomputed when a symptomatic assessment is created/updated whereas CE is not affected by any factors outside of the LDE/CE logic.

Reply:
>NSH can (and should) be recomputed when a symptomatic assessment is created/updated

Right, but I'm just saying it should also be recomputed when a Symptom Onset Date is entered, shouldn't it?

>I'm wondering if the PO team would want to notify the user that 'No Symptom History' is a field that is turned off when SO date is manually populated by a user.

Yeah that's a good thought, definitely worth checking with them.

>My immediate thought would be to make the behavior consistent/similar to how the LDE/CE logic is handled and the last time I checked, that logic is handled on the front end. However, I do think it would make more sense for this to be handled/validated on the back end as well.

Fair enough, I would say implement this change in the front end for now, and I created a ticket to add those side effects to the back end: [SARAALERT-1409](https://tracker.codev.mitre.org/browse/SARAALERT-1409).

Reply:
You're right, it is now updated here: https://github.com/SaraAlert/SaraAlert/pull/747/commits/e2c9d22524075aca7ef71f466ed78527bb6f324d#diff-168728b2f593972c872dcf3f12bcd637c7d2549ea3f8c20ffa2f5bbdab73ca4bR524

I will follow up with the PO team for language

Reply:
this is no longer relevant as `no_symptom_history` is not actually a field anymore